cves/CVE-2004-0809.yml

```diff
@@ -119,8 +122,8 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
+    - commit: 32987a707bea265cf56cbe309275f357a873991c
+      note:  It seems like the way locking was designed in this system has been very problamatic for quite some time
```

Comment:
The note for this interesting commit does not really illustrate why it was interesting.  A little more explanation as to the contents of the commit would help for ease of information access.